prow/cmd/hmac/fakeghhook/fakeghhook.go

```diff
@@ -169,10 +169,8 @@ func deleteHook(m map[string][]github.Hook, key string, req github.HookRequest)
 		return nil
 	}
 
-	hookExists := false
 	for i, hook := range m[key] {
 		if hook.Config.URL == req.Config.URL {
-			hookExists = true
```

Comment:
We always `return` when we enter this condition, so it is impossible for any other code go see the `hookExists` variable being true